src/parser/core/Combatant.ts

```diff
@@ -74,7 +74,12 @@ class Combatant extends Entity {
     if (!this.owner.characterProfile) {
       return null;
     }
-    const raceId = this.owner.characterProfile.race;
+    let raceId = this.owner.characterProfile.race;
+    if (raceId === null) {
+      // When it is an anonymous report we won't have any race.
```

Comment:
Isn't `characterProfile` always falsey already for anonymous reports?

Reply:
The object exists but many of the fields are null or empty.